charts/v3.0/cray-hms-hbtd/values.yaml

```diff
@@ -0,0 +1,110 @@
+# Please refer to https://stash.us.cray.com/projects/CLOUD/repos/cray-charts/browse/stable/cray-service/values.yaml?at=refs%2Fheads%2Fmaster
```

Comment:
Is this now the url to go to?
https://github.com/Cray-HPE/base-charts/blob/master/kubernetes/cray-service/values.yaml